data/json/professions.json

```diff
@@ -3718,4 +3718,17 @@
       "female": [ "chestwrap" ]
     }
   }
+  {
+    "type": "profession",
+    "ident": "rancher",
+    "name": "Rancher",
+    "description": "You've raised cows or horses most your life, now, we'll see what happenes next.",
```

Comment:
```suggestion
    "description": "You've raised cows or horses most your life, now, we'll see what happens next.",
```

Reply:
Most of your life.

Reply:
I thought the missing "of" was an attempt to sound in-character, so I didn't bother to fix that one. 😛